packages/cms-cli/commands/schema/create.js

```diff
@@ -0,0 +1,82 @@
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cms-lib/lib/urls');
+const { getCwd } = require('@hubspot/cms-lib/path');
+
+const {
+  validatePortal,
+  getAbsoluteFilePath,
+  isFileValidJSON,
+} = require('../../lib/validation');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const {
+  addConfigOptions,
+  addPortalOptions,
+  addTestingOptions,
+  setLogLevel,
+  getPortalId,
+} = require('../../lib/commonOpts');
+const { ENVIRONMENTS } = require('@hubspot/cms-lib/lib/constants');
+const { shouldIgnoreFile } = require('@hubspot/cms-lib/ignoreRules');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { createSchema } = require('@hubspot/cms-lib/api/schema');
+
+const action = async (args, options) => {
+  const { definition } = args;
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('schema-create', null, portalId);
+
+  const filePath = getAbsoluteFilePath(definition);
+  if (!filePath || !isFileValidJSON(filePath)) {
+    process.exit(1);
+  }
+
+  if (shouldIgnoreFile(filePath, getCwd())) {
+    logger.error(
+      `The file "${definition}" is being ignored via an .hsignore rule`
+    );
+    return;
+  }
```

Comment:
I don't think we should leverage `shouldIgnoreFile` here because it would be natural to want to avoid uploading the schemas via `.hsignore`.

Reply:
Ah yeah that makes sense.  Removing